dnf/rpm/miscutils.py

```diff
@@ -36,6 +36,8 @@ def checkSig(ts, package):
     try:
         hdr = ts.hdrFromFdno(fdno)
     except rpm.error as e:
+        # This first entry smells like a typo and is one, but don't try to "fix" it,
+        # since we're actually checking data returned by the Python RPM bindings here.
```

Comment:
This checks against an exception text from RPM, the typo was fixed in RPM 4.9 (the commit, 6834d354, dates 2009), so I'm reasonably sure we can actually remove this if?

Reply:
Yes, I think it's safe to remove. Even in RHEL 7 we have rpm 4.11. This code came from the yum times.

Reply:
ACK.

dnf itself even depends on fixed rpm versions, so there's no need to keep it around. Removed in a separate commit.